js/lib/tree.js

```diff
@@ -36,7 +31,6 @@ var NodeModel = widgets.WidgetModel.extend({
 
     initialize: function() {
         NodeModel.__super__.initialize.apply(this, arguments);
-
```

Comment:
Nitpicking here, but do you mind not removing those empty lines? IMO it improves readability.